### ISSUE TITLE
This is awesome

### ISSUE THREAD

**Issue by terrycain** [redaxmedia/razer-chroma-feedback#3]
First off this is awesome, am annoyed I didn't think of it first ;-)

Just in case you wanted to expand to other devices, `device.fx.has('static')` will return true if it supports static, as some mice and devices dont always support the same thing sadly.

Its nice to see some good applications of the python library :smile:

**Comment by redaxmedia** [comment]
Thank you very much,

Is there a way to fix the `device.fx.has('static')` issue, because the colour of my mouse is not changing :(

However, I improved the code a bit and pushed it to the develop branch:

https://github.com/redaxmedia/razer-chroma-feedback/blob/develop/bin/feedback#L38-L43

**Comment by terrycain** [comment]
Eventually yes, I have some code which sorta works out equivalent effects when syncing which if extended would be one way to solve the issue. 

The issue is that your mouse has a logo LED and a scroll LED so until I add effect endpoints which apply to all LEDs on the mouse, you can probably do something like `device.fx.has('logo_static')` then `device.fx.misc.logo.static(0, 255, 0)` and the same with scroll

**Comment by redaxmedia** [comment]
Thank you for the hint, will take a look into that tomorrow.

**Comment by redaxmedia** [comment]
Terry,

could you help me to get the experimental version inside the develop branch working?

```
ValueError: attempted relative import beyond top-level package
```

**Comment by terrycain** [comment]
Ok its basically to do with how python likes to lay out its files. You have a few options really:
1. move core.py into the bin folder and `import core`
2. make a `chroma_feedback` directory in bin, move core.py to `chroma_feedback/__init__.py` and `import chroma_feedback`
3. Sorta extends from 2, you can use `setup.py` with setuptools to deal with feedback.py being a binary and will put it in usr/bin whilst core will be put in the python library path so you can import it.

Make sense? otherwise i'm happy to PR option 2/3

**Comment by redaxmedia** [comment]
Thank you for the feedback, I try the second option yesterday but it doesn't work with `src` as a directory and empty `__init__.py` file.

**Comment by redaxmedia** [comment]
@terrycain Sorry, but we need a API documentation for the Device Manager. It took me 30 minutes to figure out that scroll_weel and not scroll is the needed sub-key to call the static function.

The [single_breath](https://github.com/redaxmedia/razer-chroma-feedback/blob/develop/src/core.py#L56) is not working at all, can you give me a helpful hand?

**Comment by terrycain** [comment]
> but we need a API documentation for the Device Manager

Its on my list, but am rather busy atm. A while ago I started on [this](https://github.com/openrazer/openrazer/tree/feature_docs) which starts to make some docs.

> Edit: single_breath was not working for misc.logo and misc.scroll_wheel so I had to switch to pulsate? why the different wording? why is there no device.fx.pulsate? :-)

the library isnt perfect ;-). The reason being is, pulsate is a rare function on only a few devices and only I think 1 keyboard out of 30. Its more commonly seen on mice, and the difference between that and breathing_single is that from my point of view are 2 completely separate hardware effects on the device. I have encountered mice which can both do pulsate and breathing_single, the effects are very similar but not identical. I'm not yet ready to go down the road and normalise the naming across devices, that will take too much time atm.
